src/Splat.Autofac.Tests/DependencyResolverTests.cs

```diff
@@ -84,5 +86,45 @@ public void AutofacDependencyResolver_Should_Resolve_Screen()
             screen.ShouldNotBeNull();
             screen.ShouldBeOfType<MockScreen>();
         }
+
+        /// <summary>
+        /// Shoulds the register Reactive UI binding type converters.
+        /// </summary>
+        [Fact]
+        public void AutofacDependencyResolver_Should_Register_ReactiveUI_BindingTypeConverters()
+        {
+            var builder = new ContainerBuilder();
+            var scheduler = RxApp.MainThreadScheduler; // invoke RxApp static constructor
+            var container = builder.Build();
+            var resolver = new AutofacDependencyResolver(container);
+            resolver.InitializeReactiveUI();
```

Comment:
If i'm not mistaken, this line make double registrations in container, because of callback, that registered in RxApp static constructor. 
https://github.com/reactiveui/ReactiveUI/blob/829e702a8ebc97c847181bb8c06f7ea04e60fe02/src/ReactiveUI/RxApp.cs#L90-L98

Reply:
Agreed on this point.

Reply:
Resolved.